docs/tutorial.rst

```diff
@@ -0,0 +1,1895 @@
+
+Finally, a test may optionally define a set of *performance functions* that will extract *figures of merit* for the test.
+These are simple test methods decorated with the :func:`@performance_function <reframe.core.builtins.performance_function>` decorator that return the figure of merit.
+In this example, we extract the ``Copy`` and ``Triad`` bandwidth values and convert them to ``float``.
+These figures of merit or *performance variables* as they are called in ReFrame's nomenclature have a special treatment:
+they are logged in the test's performance logs and a reference value per system may also be assigned to them.
+If that reference value is not met within some user-defined thresholds, the test will fail.
+
+
+Running a test
+==============
+
+Running our test is very straightforward:
+
+.. code-block:: bash
+
+   cd reframe-examples/tutorial
+   reframe -c stream/stream_runonly.py -r
+
+
+The :option:`-c` option defines the *check path* and it can be specified multiple times.
+It specifies the locations, directories or files, where ReFrame will try to look for tests.
+In this case, we simply pass the path to our test file.
+The :option:`-r` option instructs ReFrame to run the selected tests:
+
+.. code-block:: console
+
+   [ReFrame Setup]
+     version:           4.5.0-dev.1
+     command:           '/usr/local/share/reframe/bin/reframe -c stream/stream_runonly.py -r'
+     launched by:       user@myhost
+     working directory: '/home/user'
+     settings files:    '<builtin>'
+     check search path: '/home/user/reframe-examples/tutorial/stream/stream_runonly.py'
+     stage directory:   '/home/user/stage'
+     output directory:  '/home/user/output'
+     log files:         '/tmp/rfm-mzynqhye.log'
+
+   [==========] Running 1 check(s)
+   [==========] Started on Mon Nov 27 20:55:17 2023
+
+   [----------] start processing checks
+   [ RUN      ] stream_test /2e15a047 @generic:default+builtin
+   [       OK ] (1/1) stream_test /2e15a047 @generic:default+builtin
+   P: copy_bw: 19538.4 MB/s (r:0, l:None, u:None)
+   P: triad_bw: 14883.4 MB/s (r:0, l:None, u:None)
+   [----------] all spawned checks have finished
+
+   [  PASSED  ] Ran 1/1 test case(s) from 1 check(s) (0 failure(s), 0 skipped, 0 aborted)
+   [==========] Finished on Mon Nov 27 20:55:25 2023
+   Log file(s) saved in '/tmp/rfm-mzynqhye.log'
+
+
+The verbosity of the output can be increasing using the :option:`-v` option or decreased using the :option:`-q` option.
+By default, a log file is generated in the system's temporary directory that contains detailed debug information.
+The :ref:`logging` section article describes how logging can be configured in more detail.
+Once a performance test finishes, its figures of merit are printed immediately using the ``P:`` prefix.
+This can be suppressed by increasing the level at which this information is logged using the :envvar:`RFM_PER_INFO_LEVEL` environment variable.
```

Comment:
```suggestion
This can be suppressed by increasing the level at which this information is logged using the :envvar:`RFM_PERF_INFO_LEVEL` environment variable.
```